src/service/standard-actions-impl.js

```diff
@@ -174,7 +174,9 @@ export class StandardActions {
         return this.handleScrollTo_(invocation);
 
       case 'goBack':
-        Services.historyForDoc(this.ampdoc).goBack();
+        Services.historyForDoc(this.ampdoc).goBack(
+          /*canPerforWindowBack*/ !!args && args['navigate']
```

Comment:
Is `args['navigate']` a boolean or a string? If string, what if I pass `navigate=false`?

Reply:
Great question! There is a full `ParserTokenizer` that will correctly discover the type as a boolean, so navigate=false works as expected. I verified this empirically since I haven't dug in to figure out how the code works :)